### PR TITLE
Fix Image View forces in image content type

### PR DESCRIPTION
**Pull request by iRohitSingh**

### CODE REVIEW

**Comment by iRohitSingh** [comment]
Fix #293

File: news/293.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix Image View forces in image content type @iRohitSingh
```
Comment (davisagli) [inline news/293.bugfix:1]:
```suggestion
Don't force aspect ratio for the Image content type view @iRohitSingh
```

**Review by davisagli — CHANGES_REQUESTED** [review]

File: news/293.bugfix (hunk 1 above)
Comment (iRohitSingh) [reply news/293.bugfix:1]: ok , thanks

Comment (iRohitSingh) [reply news/293.bugfix]: Done .

**Review by davisagli — APPROVED** [review]
LGTM. @sneridagh @steffenri This is what we always intended for the image view, right?

**Comment by steffenri** [comment]
@davisagli Yes. Only took us like half a year to fix it :D

**Pull request merged**